app/models/patient.rb

```diff
@@ -555,42 +559,36 @@ def send_assessment(force = false)
                   continuous_exposure ||
                   dependents_exclude_self.where('monitoring = ? OR continuous_exposure = ?', true, true).exists?
 
-    # If force is set, the preferred contact time will be ignored
-    unless force
+    # Determine if it is yet an appropriate time to send this person a message.
+    unless send_now
```

Comment:
Given that we want to move a lot of the above logic out of `send_assessment` and already have a ticket for writing those tests separately, it'd be good if tests could be included as part of this PR for the actual assessment sending logic relating to the preferred contact time and method that you've refactored below.

Reply:
Which ticket is that? I definitely don't mind combining that work into this ticket instead of having that done when most of this is migrated to SQL. Those tests will likely need to be removed after that change is made, but I suppose it might make it easier to do a before and after comparison. I'm really not trying to change things with this, I'm just trying to clean up some things that have become messy and confusing.

Reply:
I'm referring to [this](https://tracker.codev.mitre.org/browse/SARAALERT-770) ticket. The ticket itself is just for writing all the tests for this logic, so that when we make changes like this we can make sure we don't accidentally/unknowingly affect any of the cases we may have forgotten about. And yes they'd likely need to be updated after the logic is moved/refactored, but we'd at least have them written and working on the existing logic.

Reply:
Going to move this PR to draft, and pick up 770.

Reply:
Sounds great!